src/tsung/tt_utils.erl

```diff
@@ -61,6 +61,26 @@ message_url({_Pid, DynVars}) ->
     ?LOGF("final url = ~p~n", [Final_Url], ?DEB),
     Final_Url.
 
+group_message_url({_Pid, DynVars}) ->
+    {ok, Ttl} = ts_dynvars:lookup(ttl, DynVars),
+    {ok, Cn} = ts_dynvars:lookup(cn_server, DynVars),
+    {ok, Organization} = ts_dynvars:lookup(messaging_org, DynVars),
+    {ok, Sender} = ts_dynvars:lookup(my_token, DynVars),
+    {ok, Session} = ts_dynvars:lookup(session_id, DynVars),
+    {ok, Cn} = ts_dynvars:lookup(cn_server, DynVars),
+    {ok, Group_Token} = ts_dynvars:lookup(group_token, DynVars),
+    Client_Id = uuid:to_string(uuid:srandom()),
+    Url = ["{", "\"body\": \"test\", ", "\"sender\":", "\"", Sender, "\",", "\"recipient\":", "\"", Group_Token, "\",",
+        "\"sender_organization\": \"", Organization, "\",", "\"recipient_organization\": \"", Organization, "\",",
+        "\"client_id\":\"", Client_Id, "\",", "\"dor\": \"0\",",
+        "\"ttl\":", "\"", Ttl, "\",", "\"xmlns\": \"tigertext:iq:group:message\",",
+        "\"status\": \"New\",", "\"ts\":", integer_to_binary(ts()), "}"],
```

Comment:
ha might be better for readability if we  write the json in erlang terms like this:

```
Doc = {[{<<"body">>, <<"test">>}, 
        {<<"sender">>, Sender},
        {<<"recipient">>, Group_Token},
        {<<"sender_organization">>, Organization},
        {<<"recipient_organization">>, Organization},
        {<<"client_id">>, Client_Id},
        {<<"dor">>, <<"0">>},
        {<<"ttl">>, Ttl},
        {<<"xmlns">>, <<"tigertext:iq:group:message">>},
        {<<"status">>, <<"New">>},
        {<<"ts">>, integer_to_binary(ts())}]},

edoc_lib:escape_uri(binary_to_list(jiffy:encode(Doc)))
```